### PR TITLE
Add Todd's event and add missing videos links

### PR DESCRIPTION
**Pull request by purna135**

### CODE REVIEW

File: data/events.toml [hunk 1]
```diff
@@ -104,7 +104,25 @@
     video1Name = "Interview video"
     video1SocialLink = "https://www.youtube.com/watch?v=acDdmyJZW-U"
     video2Name = "Event video"
-    video2SocialLink = ""
+    video2SocialLink = "https://youtu.be/ElfToZ9EBpM"
```
Comment (reshamas) [inline data/events.toml:107]: @purna135 
The thumbnail for Todd's interview is labeled "Sandra Meneses"

**Comment by reshamas** [comment]
@purna135
a) The thumbnail for Todd's interview is labeled "Sandra Meneses"
b) We should remove "Mistplay" as a sponsor because they have been listed for their 3 events already. 

We may update the event thumbnail later once the other sponsors' payments have been received. For now, we can leave that area without a sponsor logo.

**Comment by reshamas** [comment]
@purna135 
Hyosub has two videos. Not sure how we can incorporate that into the event card on the website.
a) pre-recorded talk
b) Q&A

File: data/events.toml (hunk 1 above)
Comment (purna135) [reply data/events.toml:107]: could you share a screenshot?

Comment (reshamas) [reply data/events.toml:107]: https://www.youtube.com/watch?v=P3GIrgYwsTI

**Comment by purna135** [comment]
Thank you, I will update it as suggested above

File: data/events.toml (hunk 1 above)
Comment (purna135) [reply data/events.toml:107]: got it, thank you

File: layouts/index.html [hunk 2]
```diff
@@ -65,7 +65,7 @@ <h2>PyMCon Events</h2>
                 <div class="swiper eventCardSlider">
                     <div class="swiper-wrapper">
                 {{ end }}
-                        {{ $events := sort $eventsData.event "eventOrder" }}
+                        {{ $events := sort $eventsData.event "eventOrder" "desc" }}
```
Comment (canyon289) [inline layouts/index.html:68]: Genius

**Review by canyon289 — APPROVED** [review]
With changes Reshma noted

**Comment by reshamas** [comment]
@purna135 
Thanks for this PR and making the changes.
I think it is ready to merge in.

Some feedback on the font on the social card for future cards: footer, www.pymcon.con
 The effects makes it harder to read the font. 
But, that may just be my vision.  

It's ready to merge in.

Screen Shot 2023-05-15 at 8 19 21 AM

**Comment by purna135** [comment]
> Some feedback on the font on the social card for future cards: footer, [www.pymcon.con](http://www.pymcon.con)

Agree, I think it's better to remove the effect completely, let me do it now

**Pull request merged**